src/app/modules/aqls/components/aqls/aqls.component.ts

```diff
@@ -23,11 +23,6 @@ import { AqlService } from 'src/app/core/services/aql/aql.service'
   templateUrl: './aqls.component.html',
   styleUrls: ['./aqls.component.scss'],
 })
-export class AqlsComponent implements OnInit {
-  constructor(private aqlCategoryService: AqlCategoryService, private aqlService: AqlService) {}
-
-  ngOnInit(): void {
-    this.aqlService.getAll().subscribe()
-    this.aqlCategoryService.getAll().subscribe()
-  }
+export class AqlsComponent {
```

Comment:
As the component doesn't use the aqlService neither the aqlCategoryService,  you can delete the imports as well.